pkg/scorecard_test.go

```diff
@@ -168,8 +180,8 @@ func TestRunScorecard(t *testing.T) {
 				t.Errorf("RunScorecard() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("RunScorecard() got = %v, want %v", got, tt.want)
+			if !cmp.Equal(got, tt.want, cmpopts.IgnoreFields(ScorecardResult{}, "Date")) { //nolint:govet
+				t.Errorf("expected %v, got %v", got, cmp.Diff(tt.want, got, cmpopts.IgnoreFields(ScorecardResult{}, "Date"))) //nolint:lll
```

Comment:
one thing you can do to reduce duplication and the line length is saving the `opt`.
```go
ignoreDate := cmpopts.IgnoreFields(ScorecardResult{}, "Date")
if !cmp.Equal(got, tt.want, ignoreDate) {
    t.Errorf("expected %v, got %v", got, cmp.Diff(tt.want, got, ignoreDate))
}
```

ps. not sure what the govet linter was for, doesn't seem to be triggering anymore? can probably be removed

Reply:
done. thanks for these suggestions and approving the PR!